notebooks/241-freevc/241-freevc.ipynb

```diff
@@ -0,0 +1,929 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    titles, srcs, tgts = [], [], []
</pre><p>do we really need this? not very convenient if we should use txt file and its lines parsing if we can put this info already in acceptable format (e.g. title is no need at all, it used only as file name for result, but we can use combination of source and target files names instead, it will be more clear for users to understand which files used)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1042/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>